src/ex3/index.js

```diff
@@ -0,0 +1,112 @@
+import { Command } from 'commander';
```

Comment:
When I am trying to run the program I get this error:
```
Cannot find package 'commander' imported from /Users/idan/Development/monday-u/monday-u-exercises-roni/src/ex3/index.js
```

I guess that it's because you run `npm install commander` instead of `npm install --save commander`. I ran it locally to make it work but please fix it

Reply:
I got this error as well:
<img width="1049" alt="image" src="https://user-images.githubusercontent.com/9549288/173353768-3070dd95-99c7-4850-a630-846b40b920eb.png">

I had to install `ascii-art-ansi` package manually. let's talk about it in the next open hour to understand better how to use npm packages and `package.json`